index.php

```diff
@@ -7,114 +7,31 @@
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
 
-// Appel des contrôleurs
-require_once "controllers/Frontend.php";
-require_once "controllers/Backend.php";
-
 // Appel d'un fichier pour l'url rewritting
 require_once "config/function_url.php";
+require_once "config/autoload.php";
 
 // Affichage de la page d'accueil lors du lancement de l'index.php
-if (!filter_has_var(INPUT_GET, 'page'))
-{
-    $controller = new Frontend();
-    $controller->home_cv();
-    require_once "views/frontend/home_cv.php";
-}
-
-// Affichage des autres page en fonction du GET
-if (filter_has_var(INPUT_GET, 'page'))
-{
-    // FRONTEND
-    switch(filter_input(INPUT_GET, 'page')){
-        case 'home_cv':
-            $controller = new Frontend();
-            $controller->home_cv();
-        break;
-    
-        case 'legalnotice':
-            $controller = new Frontend();
-            $controller->legalnotice();
-        break;
-    
-        case 'home':
-            $controller = new Frontend();
-            $controller->home();
-        break;
-    
-        case'blog':
-            $controller = new Frontend();
-            $controller->blog();
-        break;
+//if (!filter_has_var(INPUT_GET, 'page'))
+//{
+//    $controller = new Frontend();
+//    $controller->home_cv();
+//    require_once "views/frontend/home_cv.php";
+//}
 
-        case 'post':
-            if(filter_has_var(INPUT_GET, 'id')){
-                $controller = new Frontend();
-                $controller->post(filter_input(INPUT_GET, 'id'));
-            }
-        break;
-        
-        case'error':
-            $controller = new Frontend();
-            $controller->error();
-        break;
-        
-    // BACKEND
-        case 'dashboard':
-            $controller = new Backend();
-            $controller->dashboard();
-        break;
 
-        case 'write':
-            $controller = new Backend();
-            $controller->write();
-        break;
 
-        case 'settings':
-            $controller = new Backend();
-            $controller->settings();
-        break;
+MyAutoload::start();
 
-        case 'list':
-            $controller = new Backend();
-            $controller->list();
-        break;
 
-        case 'logout':
-            $controller = new Backend();
-            $controller->logout();
-        break;
+// Affichage des autres page en fonction du GET
 
-        case 'new':
-            $controller = new Backend();
-            $controller->new();
-        break;
 
-        case 'new_admin':
-            $controller = new Backend();
-            $controller->new();
-        break;
-    
-        case 'password':
-            $controller = new Backend();
-            $controller->password();
-        break;
+$request = $_GET['page'];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Direct use of $_GET Superglobal detected.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)